consensus/ssz_derive/src/lib.rs

```diff
@@ -573,6 +597,67 @@ fn ssz_encode_derive_enum_transparent(
     output.into()
 }
 
+/// Derive `ssz::Encode` for an `enum` following the "tag" SSZ spec.
```

Comment:
Sorry, the confusion here is due to me copying the comment below:
> /// Derive `ssz::Encode` for an `enum` following the "union" SSZ spec.

Instead of copying the comment above:
> /// Derive `ssz::Encode` for an `enum` in the "transparent" method.

I didn't mean to imply there was a `spec`. Of course there wouldn't be. What we're talking about here are ways of translating a rust `enum` into something that can be represented in `ssz`. We have two *methods* thus far:
1. `transparent` (to ignore the `enum`
2. `union` for rust `enum`'s where each variant has a single field.

There is no easy way of representing rust `enum`s where variants have more than one field or varying numbers of fields. But it's quite natural to have a *method* of representing rust `enum`s where all variants have **no** fields. I've taken to calling this *method* `tag` because we're "tagging" each variant with a unique `u8`.

Reply:
Ah ok, cool good enough for me! Seems useful